src/main/java/com/bookappstore/controller/ShoppingCartController.java

```diff
@@ -0,0 +1,81 @@
+package com.bookappstore.controller;
+
+import com.bookappstore.dto.cart.CartItemRequestDto;
+import com.bookappstore.dto.cart.CartItemUpdateDto;
+import com.bookappstore.dto.cart.ShoppingCartResponseDto;
+import com.bookappstore.model.User;
+import com.bookappstore.service.ShoppingCartService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import java.awt.print.Pageable;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Sort;
+import org.springframework.data.web.SortDefault;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.security.core.Authentication;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/cart")
```

Comment:
Use server.servlet.context-path=/api
```suggestion
@RequestMapping("/cart")
```